docs/UserGuide.md

```diff
@@ -237,6 +237,13 @@ Clears all book and user entries from BookFace.
 
 Format: `clear all`
 
+### Changing the theme: `theme` *COMING SOON*
```

Comment:
I know it says **__`COMING SOON`__** but our project constraint is that CLI > GUI, so the `theme light` feature should be implemented before the clicking of theme buttons.